packages/core/src/model-manager.ts

```diff
@@ -1,41 +1,49 @@
 'use strict';
 
 import defaults from 'lodash/defaults';
-
-const Toposort = require('toposort-class');
+import Toposort from 'toposort-class';
+import type { ModelStatic } from './model';
+import type { Sequelize } from './sequelize';
 
 export class ModelManager {
-  constructor(sequelize) {
+  declare sequelize: Sequelize;
```

Comment:
you don't need to use the `declare` keyword here, they're actual properties

This may also be a good opportunity to make these two fields `#`-private? `#sequelize`

Reply:
done